test/Graphics/RenderWindow.test.cpp

```diff
@@ -1,8 +1,60 @@
+#include <SFML/Graphics/Image.hpp>
 #include <SFML/Graphics/RenderWindow.hpp>
+#include <SFML/Graphics/Texture.hpp>
+#include <SFML/System/String.hpp>
+#include <SFML/Window/VideoMode.hpp>
 
+#include <doctest/doctest.h>
+
+#include <GraphicsUtil.hpp>
 #include <type_traits>
 
 static_assert(!std::is_copy_constructible_v<sf::RenderWindow>);
 static_assert(!std::is_copy_assignable_v<sf::RenderWindow>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::RenderWindow>);
 static_assert(!std::is_nothrow_move_assignable_v<sf::RenderWindow>);
+
+TEST_CASE("[Graphics] sf::RenderWindow" * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        sf::RenderWindow window(sf::VideoMode(sf::Vector2u({256, 256}), 24),
+                                "Window Title",
+                                sf::Style::Default,
+                                sf::ContextSettings());
+
+        CHECK(window.getSize() == sf::Vector2u({256, 256}));
+    }
+
+    SUBCASE("Clear")
+    {
+        sf::RenderWindow window(sf::VideoMode(sf::Vector2u({256, 256}), 24),
+                                "Window Title",
+                                sf::Style::Default,
+                                sf::ContextSettings());
+
+        REQUIRE(window.getSize() == sf::Vector2u({256, 256}));
+
+        sf::Texture texture;
+
+        REQUIRE(texture.create(window.getSize()));
+
+        window.clear(sf::Color::Red);
+
+        texture.update(window);
+
+        CHECK(texture.copyToImage().getPixel(sf::Vector2u({64, 64})) == sf::Color::Red);
+
+        window.clear(sf::Color::Green);
+
+        texture.update(window);
+
+        CHECK(texture.copyToImage().getPixel(sf::Vector2u({128, 128})) == sf::Color::Green);
+
+        window.clear(sf::Color::Blue);
+
+        texture.update(window);
+
+        CHECK(texture.copyToImage().getPixel(sf::Vector2u({196, 196})) == sf::Color::Blue);
```

Comment:
We can not double-space our code? It doubles the line count with aiding in readability in my opinion. There are some other places in this PR where I could say the same.